core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1942,7 +1952,8 @@ TimeoutException coordinatorTimeoutWaitingForView(int expectedViewId, int curren
    @Message(value = "Unexpected response from %s: %s", id = 570)
    IllegalArgumentException unexpectedResponse(Address target, Response response);
 
-   @Message(value = "RELAY2 not found in ProtocolStack. Unable to do cross-site requests.", id = 571)
+   @Message(value = "RELAY2 not found in the protocol stack. Cannot perform cross-site operations.", id = 571)
+   @Description("To back up caches from one site to another the cluster transport uses the JGroups RELAY2 protocol. Add RELAY2 to your cluster transport configuration.")
```

Comment:
Yeah, maybe. That's one more place where we'd need to maintain a link though. And we'd have to have the community link and the downstream link.

We do this for the console though. I wonder if it would make sense to add variables to `build/configuration/pom.xml`?